packages/host/app/components/matrix/room-message.gts

```diff
@@ -308,6 +310,24 @@ export default class RoomMessage extends Component<Signature> {
       .join(', ');
   }
 
+  //TODO: Will remove this
+  get isPatchCommand() {
+    if (!this.args.message.command) {
+      return false;
+    }
+    return this.args.message.command.commandType === 'patchCard';
+  }
+
+  get isCommand() {
+    if (!this.args.message.command) {
+      return false;
+    }
+    return (
+      this.args.message.command.commandType === 'patchCard' ||
+      this.args.message.command.commandType === 'searchCard'
+    );
+  }
+
```

Comment:
we need to think thru solutions for command specific logic to not get aggregated into the same module. I think we should be asking our selves: "what is the procedure for adding a new command". if the answer is: "you need to modify these 6 different files to add your command logic", I think that is the wrong answer.